contents/blog/posthog-vs-statsig.mdx

```diff
@@ -0,0 +1,294 @@
+
+On the other hand, PostHog offers [multivariate flags with JSON payloads](/docs/feature-flags/creating-feature-flags#served-value) (in addition to boolean flags). This provides greater flexibility in testing and deploying different variations of a feature, making them more suitable for complex rollouts.
+
+<ComparisonTable column1="PostHog" column2="Statsig">
+  <ComparisonRow column1={true} column2={true} feature="Boolean flags" description="Simple flags returning true or flag" />
+  <ComparisonRow column1={true} column2={false} feature="Multivariate flags" description="Flags with multiple customizable values" />
+  <ComparisonRow column1={true} column2={false} feature="Payloads" description="Flags with string, number, or JSON payloads" />
+  <ComparisonRow column1={true} column2={true} feature="Percentage rollouts" description="Target percentages of a group" />
+  <ComparisonRow column1={true} column2={true} feature="Custom targeting" description="Target users based on user properties, custom contexts" />
+  <ComparisonRow column1={false} column2={true} feature="Scheduling" description="Schedule flags to turn on or off" />
+  <ComparisonRow column1="Partial" column2={true} feature="Environments" description="Manage flags for dev, stage, prod" />
+  <ComparisonRow column1={true} column2={true} feature="Bootstrapping" description="Flags available on frontend application load" />
+  <ComparisonRow column1={true} column2={false} feature="Early access management" description="Manage betas, test features" />
+</ComparisonTable>
+
+### Product analytics
+
+Both PostHog and Statsig offer the functionality you expect from product analytics tools, such as dashboards, graphs, and funnels. However:
+
+- **PostHog** provides deep insights into how users are interacting with your product. It includes features such as lifecycle, stickiness, correlation, and retention analysis.
+
+- **Statsig** does provide some additional insights, such as retention analysis, but its main focus is to use product analytics to set up metrics to run A/B tests on. 
+
+<ComparisonTable column1="PostHog" column2="Statsig">
+  <ComparisonRow column1={true} column2={false} feature="Autocapture" description="Capture events without manual logging" />
+  <ComparisonRow column1={true} column2={true} feature="Dashboards" description="Combine insights into shareable dashboards" />
+  <ComparisonRow column1={true} column2={true} feature="Graphs and trends" description="Build custom insights and visualizations" />
+  <ComparisonRow column1={true} column2={true} feature="Cohorts" description="Combine users based on properties and events for group analysis" />
+  <ComparisonRow column1={true} column2={true} feature="Group analytics" description="Track metrics at a company level" />
```

Comment:
```suggestion
  <ComparisonRow column1={true} column2={false} feature="Group analytics" description="Track metrics at a company level" />
```

Reply:
No mention of group or account-level analytics I can find.

Reply:
They do, they call it CustomID though. 

https://docs.statsig.com/client/concepts/user
https://docs.statsig.com/messages/serverRequiredUserID
https://docs.statsig.com/guides/experiment-on-custom-id-types